src/hookfs/mod.rs

```diff
@@ -29,7 +29,7 @@ use nix::unistd::{
 
 use tracing::{debug, error, instrument, trace};
 
-use std::{collections::{HashMap, LinkedList}};
+use std::collections::{HashMap, LinkedList};
```

Comment:
These imports are chaotic, could you fix them in this PR? We should import them into three groups like following:

```rust
use std::collections::{HashMap, LinkedList};
...(other imports from standard crates)
<blank line>
use tracing::{debug, error, instrument, trace};
...(other imports from third-party crates)
<blank line>
use runtime::spawn_blocking;
...(other imports from current crate)
```

Reply:
I wonder if there is a "rustuses" like "goimports" to sort imports. I'll look into it.

Reply:
Good, there's a nightly feature in rustfmt for this